base/bitarray.jl

```diff
@@ -1301,6 +1301,23 @@ function (|)(B::BitArray, x::Bool)
 end
 (|)(x::Bool, B::BitArray) = B | x
 
+"""
+    xor(x, y)
+    ⊻(x, y)
+
+Bitwise exclusive or of `x` and `y`.  The infix operation
+`a ⊻ b` is a synonym for `xor(a,b)`, and
+`⊻` can be typed by tab-completing `\\xor`
+or `\\veebar` in the Julia REPL.
+
+```jldoctest
+julia> [true; true; false] ⊻ [true; false; false]
+3-element Array{Bool,1}:
+ false
+  true
+ false
+```
+"""
```

Comment:
Sorry for the delay! Do you think attaching this docstring to `xor(x::Bool, y::Bool)` in bool.jl is reasonable?